legate/tester/stages/_linux/gpu.py

```diff
@@ -50,7 +50,7 @@ def __init__(self, config: Config, system: TestSystem) -> None:
         self._init(config, system)
 
     def env(self, config: Config, system: TestSystem) -> EnvDict:
-        return dict(CUNUMERIC_TEST_ENV)
```

Comment:
Not immediately relevant to this PR, but shouldn't this also be using UNPIN_ENV?

Reply:
I'm not sure cc @magnatelee